pkg/controller/cluster/main-controller.go

```diff
@@ -1410,6 +1485,30 @@ func (c *Controller) enqueueTenant(obj interface{}) {
 	c.workqueue.AddRateLimited(key)
 }
 
+// enqueuePolicyBinding takes a PolicyBinding resource and converts it into a namespance/name string
+// This key is put into the workqueue.
+// It will ignore any PolicyBinding not in the namespaces that the Operator watches.
+// Only PolicyBindings in the watched namespaces where Operator manage tenants are Honored.
+func (c *Controller) enqueuePB(obj interface{}) {
```

Comment:
changes to policy bindings keep getting queued here, but nothing dequeues them ever, so this will cause a memory leak, we should remove it

Reply:
removed